packages/account/src/Components/poi/status/unsupported/detail-component.tsx

```diff
@@ -63,11 +63,11 @@ const DetailComponent = ({
                 const expiration_date =
                     typeof data.expiry_date?.format === 'function' ? data.expiry_date.format('YYYY-MM-DD') : undefined;
                 uploadFile(file, WS.getSocket, {
-                    documentType: document_type,
-                    pageType,
-                    expirationDate: expiration_date,
-                    documentId: data.document_id || '',
-                    lifetimeValid: +(lifetime_valid && !expiration_date),
+                    document_type,
+                    page_type: pageType,
+                    expiration_date,
+                    document_id: data.document_id || '',
+                    lifetime_valid: +(lifetime_valid && !expiration_date),
```

Comment:
clarification: why do we set `lifetime_valid` to a number? shouldnt it a be a boolean?

Reply:
API accepts 1 or 0 as value for that prop

Reply:
might be cool to explicitly return 1 or 0 - my firs reaction to this line is to change it from "+(lifetime_valid && !expiration_date)" to "!+(lifetime_valid && !expiration_date)"